plugins/lookup/dig.py

```diff
@@ -139,83 +142,83 @@
     msg: "{{ lookup('community.general.dig', 'example.org./A', retry_servfail=true) }}"
 """
 
-RETURN = """
-  _list:
-    description:
-      - List of composed strings or dictionaries with key and value
-        If a dictionary, fields shows the keys returned depending on query type
-    type: list
-    elements: raw
-    contains:
-       ALL:
-           description:
-               - owner, ttl, type
-       A:
-           description:
-               - address
-       AAAA:
-           description:
-               - address
-       CAA:
-           description:
-               - flags
-               - tag
-               - value
-           version_added: 6.3.0
-       CNAME:
-           description:
-               - target
-       DNAME:
-           description:
-               - target
-       DNSKEY:
-            description:
-                - flags, algorithm, protocol, key
-       DS:
-            description:
-                - algorithm, digest_type, key_tag, digest
-       HINFO:
-            description:
-                -  cpu, os
-       LOC:
-            description:
-                - latitude, longitude, altitude, size, horizontal_precision, vertical_precision
-       MX:
-            description:
-                - preference, exchange
-       NAPTR:
-            description:
-                - order, preference, flags, service, regexp, replacement
-       NS:
-            description:
-                - target
-       NSEC3PARAM:
-            description:
-                - algorithm, flags, iterations, salt
-       PTR:
-            description:
-                - target
-       RP:
-            description:
-                - mbox, txt
-       SOA:
-            description:
-                - mname, rname, serial, refresh, retry, expire, minimum
-       SPF:
-            description:
-                - strings
-       SRV:
-            description:
-                - priority, weight, port, target
-       SSHFP:
-            description:
-                - algorithm, fp_type, fingerprint
-       TLSA:
-            description:
-                - usage, selector, mtype, cert
-       TXT:
-            description:
-                - strings
+RETURN = r"""
+_list:
+  description:
+    - List of composed strings or dictionaries with key and value If a dictionary, fields shows the keys returned depending
+      on query type.
+  type: list
+  elements: raw
+  contains:
+    ALL:
+      description:
+        - Owner, ttl, type.
```

Comment:
```suggestion
        - owner, ttl, type.
```

Reply:
@felixfontein a second opinion, if you may: should I use C() or O(ignore: ..) here? These entries are actually (or seem to be) the keys of the resulting dictionary (a dict in a list, as it seems).

Reply:
It should be either `C(owner)` or `RV(ignore:list[].owner)`, since that's the full return value name. (`owner` is just a key inside a dictionary.) I'd use `C(...)`.